server/chat-commands/core.ts

```diff
@@ -1103,13 +1132,15 @@ exports.commands = {
 			return this.sendReply(`The game timer is ON (requested by ${[...timer.timerRequesters].join(', ')})`);
 		}
 		const force = user.can('timer', null, room);
-		if (!force && !room.game.playerTable[user]) {
+		if (!force && !room.game.playerTable[user.id]) {
 			return this.errorReply(`Access denied.`);
 		}
 		if (this.meansNo(target) || target === 'stop') {
 			if (timer.timerRequesters.size) {
-				timer.stop(force ? undefined : user);
-				if (force) room.send(`|inactiveoff|Timer was turned off by staff. Please do not turn it back on until our staff say it's okay.`);
+				const staffForced: any = force ? undefined : user;
+				timer.stop(staffForced);
```

Comment:
Who taught you to write code like this?

Reply:
me, which would be the problem - wasn't sure how to handle it and this worked :s